### PR TITLE
tools: "Assets/web3/coin/presale/admin" and "Assets/web3/coin/presale/buy"

### PR DESCRIPTION
**Pull request by artman325**
Example to initiate
```
echo Q::tool("Assets/web3/coin/presale/buy", array(
        "chainId" => "0x13881",
        //"showShortInfo" => true,
        //"fund" => '0xcebbe553d62E4710BaAc52666D749Bbc35f40428'
)) 
```
P.S. this pull request is a continued of previous pull request https://github.com/Qbix/Platform/pull/2108

### CODE REVIEW

File: platform/plugins/Assets/text/Assets/content/en.json [hunk 1]
```diff
@@ -626,6 +626,99 @@
...
+						"sellingToken": "Selling Token",
+						"timestamps": "Untiltime",
+						"prices": "ETH, Price",
+						"endTime": "End Time",
+						"thresholds": "token, Threshold",
+						"bonuses": "% Bonus",
+						"ownerCanWithdraw": "Owner Can Withdraw",
+						"whitelistData": "whitelist Data",
+						"communityAddress": "Community address",
+						"methodkeccack": "4bytes hash of method to check",
```
Comment (EGreg) [inline platform/plugins/Assets/text/Assets/content/en.json]: please fix the language files to not have this kind of stuff, it's only for user-facing strings that make it into the final UX

File: platform/plugins/Assets/text/Assets/content/en.json [hunk 2]
```diff
@@ -626,6 +626,99 @@
...
+						"roleId": "index of role id"
```
Comment (EGreg) [inline platform/plugins/Assets/text/Assets/content/en.json]: ID not id, in user-facing

File: platform/plugins/Assets/text/Assets/content/en.json [hunk 3]
```diff
@@ -626,6 +626,99 @@
...
+					},
+					"fields": {
+						"ownerWithdrawOptions": {
+							"option0":"Owner can not withdraw",
```
Comment (EGreg) [inline platform/plugins/Assets/text/Assets/content/en.json]: put "can't" everywhere you say "can not"

**Review by EGreg — CHANGES_REQUESTED** [review]
See comments, it's about text files

File: platform/plugins/Assets/text/Assets/content/en.json (hunk 2 above)
Comment (artman325) [reply platform/plugins/Assets/text/Assets/content/en.json]: fixed

File: platform/plugins/Assets/text/Assets/content/en.json (hunk 1 above)
Comment (artman325) [reply platform/plugins/Assets/text/Assets/content/en.json]: fixed

File: platform/plugins/Assets/text/Assets/content/en.json (hunk 3 above)
Comment (artman325) [reply platform/plugins/Assets/text/Assets/content/en.json]: done

**Review by EGreg — APPROVED** [review]

**Pull request merged**